pkg/lib-volume-populator/populator-machinery/controller.go

```diff
@@ -586,13 +588,21 @@ func (c *controller) syncPvc(ctx context.Context, key, pvcNamespace, pvcName str
 				// Join the transfer network namespace and name
 				annotations[AnnDefaultNetwork] = fmt.Sprintf("%s/%s", transferNetwork["namespace"], transferNetwork["name"])
 			}
-
+			migration, found, err := unstructured.NestedString(crInstance.Object, "metadata", "labels", "migration")
+			if err != nil {
+				return err
+			}
+			labels := map[string]string{"pvcName": pvc.Name}
```

Comment:
@bennyz we can leave it or drop it. we don't set it once the pod exists and don't have it. i am in favor of leaving it.